libs/design/src/molecules/sidebar/sidebar/sidebar.component.scss

```diff
@@ -1,8 +1,37 @@
+@use '../scss/helpers';
+
 :host {
 	display: block;
-}
+	width: 300px;
```

Comment:
Width should also be determined by the user.

Reply:
I think we should specify a sane default.

Reply:
I could use an assist here specifying nice colors, however :)